Level-0/00-Tools-of-the-Trade/The-Browser.md

```diff
@@ -56,7 +56,7 @@ website.
   parts are when using a URL. Here they are:
 
     * `http` or `https`
-        - This means Hyper Text Transfer Protocol or Hyper Text Transfer Protocol . A Protocol is a set of rules to be
+        - This means Hyper Text Transfer Protocol or Hyper Text Transfer Protocol Secure. A Protocol is a set of rules to be
```

Comment:
@daveissomewhere This is more correct that what was there but is also technically incorrect. HTTPS stands for HTTP over SSL. However, sine we aren't really using any of SSLv1, SSLv2, SSLv3 anymore due to inherit security vulnerabilities, I can see where this misduplication comes from. Wikipedia is also get is wrong on this one, perpetuating the misduplication; they like to rewrite history.

Since this is meant to be bring new people up to speed on technology, it would be okay to try to make it simpler. I also think that we should be correct and accurate. We would also need to make sure the SSL is defined here.

Reply:
Ok, yeah, I wasn't aware of that.  Off to the learning board with me. :-)